packages/cli/src/commands/transfer/ExportCommand.ts

```diff
@@ -17,6 +17,7 @@ type Options = {
 	'no-gzip-output'?: boolean
 	'no-gzip-transfer'?: boolean
 	output?: string
+	'ignore-table'?: string[]
```

Comment:
you have `include-system` option, so consider using `exclude` instead of `ignore`